src/nwbinspector/checks/time_series.py

```diff
@@ -88,6 +88,21 @@ def check_timestamps_ascending(time_series: TimeSeries, nelems=200):
         return InspectorMessage(f"{time_series.name} timestamps are not ascending.")
 
 
+@register_check(importance=Importance.BEST_PRACTICE_VIOLATION, neurodata_type=TimeSeries)
```

Comment:
I'm unsure if this is serious enough to merit a `VIOLATION`. It doesn't cause any serious issues within the schema (as long as the time reference are, in truth, synchronized to be relative to the session start time) or external tools (AFAIK). It's mostly a flavor and 'expectation' thing to avoid confusion for a first time user upon seeing a negative value for time (has to think about what that means; if always positive, it's pretty clear).

I'd say maybe just a `SUGGESTION` is fine

Reply:
Changed it to **BEST_PRACTICE_SUGGESTION**